models/Store.js

```diff
@@ -4,7 +4,7 @@ const mongoose = require('mongoose');
 
 const schema = new mongoose.Schema({
   name: { type: String, required: true },
-  address: { type: String, required: true },
+  address: { type: Object, required: true },
   googleId: { type: String, required: true, unique: true },
```

Comment:
Wouldn't this be a good time to switch away from the googleId name?

Reply:
Probably, I'll change that over alongside some of the routing for stores